README.md

```diff
@@ -2,13 +2,15 @@
 
 # LaTeX document class for Quantum
 
-This is version 5.1 of `quantumarticle`, the document class for typesetting articles in Quantum - the open journal for quantum science.
+This is version 6.0 of `quantumarticle`, the document class for typesetting articles in Quantum - the open journal for quantum science.
```

Comment:
Good catch!

Reply:
Wait. I actually don't get why this is shown here. This change is already in both the `develop` and the `nolink_error` branch.

Reply:
I'm quite confused by the entire diff that GitHub is showing. I checked out the develop branch, edited only `quantum.bst` and changed the base branch of this PR to `develop`, but 15 files show diffs.. Maybe it's easiest if I retry manually.